ui/src/serverIO.js

```diff
@@ -109,8 +109,8 @@ class ServerIO {
 
   disconnect() {
     this.connected = false;
-    this.hwrSocket.close();
-    this.loggingSocket.close();
+    this.hwrSocket?.close();
+    this.loggingSocket?.close();
```

Comment:
Logging out with `VITE_DISABLE_WEBSOCKETS=true` was throwing an error. I'll do a better refactor of this file next.